sources/packages/backend/workflow/test/2022-2023/assessment-gateway/assessment-gateway.originalAssessment.e2e-spec.ts

```diff
@@ -57,4 +78,227 @@ describe(`E2E Test Workflow assessment gateway on original assessment for ${PROG
       WorkflowServiceTasks.UpdateNOAStatusToRequired,
     );
   });
+
+  it("Should check for both parents incomes when the student is dependant and parents have SIN.", async () => {
+    // Arrange
+    const currentAssessmentId = assessmentId++;
+    const parent1SupportingUserId = supportingUserId++;
+    const parent2SupportingUserId = supportingUserId++;
+    // Assessment consolidated mocked data.
+    const assessmentConsolidatedData: AssessmentConsolidatedData = {
+      assessmentTriggerType: AssessmentTriggerType.OriginalAssessment,
+      ...createFakeConsolidatedFulltimeData(PROGRAM_YEAR),
+      ...createParentsData({ numberOfParents: 2 }),
```

Comment:
I know this way works, but the supporting user parent data must be returned by the following worker and not the load-assessment-consolidated data right?

![image](https://user-images.githubusercontent.com/54600590/229626488-57bf83f2-b8e2-433b-8ee8-45d45971fbcc.png)

Reply:
I believe that there is confusion here. The service task above has specific mocked data to return its data.
The assumption that parent data is returned by the workflow in the picture is not correct.
All the data (I mean, all the data) to process an assessment (and reassessments) is returned in the consolidated data.
Keep in mind that the reassessment does not go down this path and if this was the workflow responsible for loading the parent's data the reassessment would never have supporting users' data.
Please let me know If it is clear or if I am missing something.